### PR TITLE
Add management VRF config CLI

### PR DESCRIPTION
**Pull request by mingjunzhang2019**
This is to add management VRF config CLI commands:

sonic(config)# ip vrf management
sonic(config)# no ip vrf management

### CODE REVIEW

File: src/CLI/actioner/sonic-cli-vrf.py [hunk 1]
```diff
@@ -21,120 +21,59 @@
...
-                    print("Success")
-                else:
-                    print ("Failed")
-            else:
+    if response.ok():
+        if response.content is not None:
+            # Get Command Output
+            api_response = response.content
+            if api_response is None:
                 print("Failed")
```
Comment (joyas-joseph) [inline src/CLI/actioner/sonic-cli-vrf.py]: Let's have some more meaningful format/message.

File: src/CLI/clitree/cli-xml/vrf.xml [hunk 2]
```diff
@@ -25,12 +25,12 @@ limitations under the License.
     <VIEW name="enable-view">
       <COMMAND name="show ip vrf" help="Show vrf information">
```
Comment (jeffyin-dell) [inline src/CLI/clitree/cli-xml/vrf.xml]: Can we capitalize "VRF" in the `help` string?

File: src/CLI/clitree/cli-xml/vrf.xml [hunk 3]
```diff
@@ -25,12 +25,12 @@ limitations under the License.
...
         <ACTION> 
-          python $SONIC_CLI_ROOT/openconfig_network_instance_api.py get_list_openconfig_network_instance_network_instances_network_instance show_mgmt_vrf.j2
+          python $SONIC_CLI_ROOT/sonic-cli-vrf.py get_openconfig_network_instance_network_instances_network_instance show_vrf.j2
         </ACTION>
       </COMMAND>
       <COMMAND name="show ip vrf management" help="Show management vrf information">
```
Comment (jeffyin-dell) [inline src/CLI/clitree/cli-xml/vrf.xml]: Capitalize VRF in the `help` string here as well, and across the entire XML file. It's important to have proper formatting and be consistent throughout. I know this doesn't fall under your diffs, but thanks for taking care of it anyway.

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]

File: src/CLI/actioner/sonic-cli-vrf.py (hunk 1 above)
Comment (mingjunzhang2019) [reply src/CLI/actioner/sonic-cli-vrf.py]: Done

File: src/CLI/clitree/cli-xml/vrf.xml (hunk 2 above)
Comment (mingjunzhang2019) [reply src/CLI/clitree/cli-xml/vrf.xml]: Done

File: src/CLI/clitree/cli-xml/vrf.xml (hunk 3 above)
Comment (mingjunzhang2019) [reply src/CLI/clitree/cli-xml/vrf.xml]: Done

**Review by jeffyin-dell — APPROVED** [review]

**Pull request merged**